Doppler.ContactPolicies.Business.Logic/Services/ContactPoliciesService.cs

```diff
@@ -26,5 +26,10 @@ public async Task UpdateContactPoliciesSettingsAsync(string accountName, Contact
             var contactPoliciesToUpdate = contactPoliciesSettings.ToDao();
             await _contactPoliciesSettingsRepository.UpdateContactPoliciesSettingsAsync(accountName, contactPoliciesToUpdate);
         }
+
+        public async Task<int?> GetIdUserByAccountName(string accountName)
+        {
+            return await _contactPoliciesSettingsRepository.GetUserIdByAccountName(accountName);
```

Comment:
What do you think about call _user id_ in the same form always?

```suggestion
        public async Task<int?> GetIdUserByAccountName(string accountName)
        {
            return await _contactPoliciesSettingsRepository.GetIdUserByAccountName(accountName);
```

Reply:
Resolved!